README.md

```diff
@@ -446,11 +448,19 @@ $storage = new StorageClient([
 
 This library follows [Semantic Versioning](http://semver.org/).
 
-Please note it is currently under active development. Any release versioned 0.x.y is subject to backwards incompatible changes at any time.
+Please note it is currently under active development. Any release versioned
+0.x.y is subject to backwards incompatible changes at any time.
+
+**GA**: Libraries defined at a GA quality level are stable, and will not
```

Comment:
Given the line above that refers to `0.x.y`, we should probably clarify here that this refers only to clients versioned at `1.0.0` and above.